src/styles/about.scss

```
@@ -0,0 +1,31 @@
+@import "_variables";
+
+.about {
+  display: flex;
+  width: 100%;
+  margin: 0;
+  justify-content: center;
+  &__left {
```

Розділяй селектори відступами, так простіше читати
```suggestion
  justify-content: center;
  
  &__left {
```